Retry delete

FreddieAkeroyd opened the pull request:
Retry delete - may or may not help...

Alexandru-Dascalu commented:
![image](https://user-images.githubusercontent.com/34415597/78568839-5b5d7600-782b-11ea-83d1-9d3c672afb0e.png)
I am getting this error when trying to run instrument_install_latest_build_only.bat on demo. The "badumtsss" word is printed when it tries a second time to delete the epics folder. But this fails again.

Alexandru-Dascalu commented:
@FreddieAkeroyd did you run that script yourself on DEMO, did it work?

FreddieAkeroyd commented:
I didn't run on demo. I made a few changes that I thought wouldn't make any difference and didn't.

I renamed the directory, just to check it wasn't been held open by another process and it wasn't, so i renamed it back and then when i ran the install script it worked. So the act of renaming the directory and renaming it back seemed to fix whatever problem there was!
